Userland/Libraries/LibGL/GLLights.cpp

```diff
@@ -34,13 +44,13 @@ void glLightModelf(GLenum pname, GLfloat param)
 
 void glLightModelfv(GLenum pname, GLfloat const* params)
 {
-    switch (pname) {
-    case GL_LIGHT_MODEL_AMBIENT:
-        g_gl_context->gl_light_model(pname, params[0], 0.0f, 0.0f, 0.0f);
-        break;
-    default:
-        g_gl_context->gl_light_model(pname, params[0], params[1], params[2], params[3]);
-    }
+    g_gl_context->gl_light_model(pname, params[0], params[1], params[2], params[3]);
+}
+
+void glLightModeliv(GLenum pname, GLint const* params)
+{
+    // Less plumbing!
```

Comment:
What?

Reply:
Whoops. This was for another function I was playing around with :sweat_smile: